fdb-relational-core/src/main/antlr/RelationalParser.g4

```diff
@@ -177,16 +177,12 @@ dropTempFunction
     : DROP TEMPORARY FUNCTION (IF EXISTS)? schemaQualifiedRoutineName=fullId
     ;
 
-createFunction
```

Comment:
`createFunction` was never used, so deleted it, as well as the method in the Visitors classes.